blog/_posts/2016-10-15-team-kindr3d-saying-bye.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Team kindr3d – Saying Bye!
+layout: post
+created_at: Tue Oct 15 2016
+permalink: blog/2016-07-19-team-kindr3d-saying-hi
+current: blog
+author: kindr3d
+twitter: TKindr3d
+categories:
+- student-posts
+---
+[Content Note: again, many animated gifs]
+
+**How much can change in just three months?**
+
+Apparently quite a lot. We wanted to tell you how in one summer we became better developers and hopefully better humans as well.
+
+![BFFs](/img/blog/2016/team-kindr3d-berlin.jpg)
+
+We feel so lucky to have spent those three months learning, communicating and building things. We suddenly became part of a bigger community of people who care about each other and are ready to give their time to help beginners. It is very humbling and empowering to feel surrounded by amazing people. But at the same time we became aware of giving back our gratitude to the this community. Also we learned the value of beginners. Our enthusiasm and drive are valuable and and needed in the world where innovation is the king.
+
+![labyrinth 1](http://i.giphy.com/10DNd65ckdlA9q.gif)*beating negativity and self doubt*
```

Comment:
please add all these gifs to the `img/blog/2016` folder as well, and add image credits for them all (i.e. giphy). This is to avoid hotlinking, which might cause a broken link/image in the future.